cypress.json

```diff
@@ -0,0 +1,7 @@
+{
+  "pageLoadTimeout": 90000,
+  "integrationFolder": "client/tests/cypress/integration",
```

Comment:
Leave only the `integrationFolder` setting. You can remove the rest of the settings. I don't think they are needed at the moment. Also, Is the `pageLoadTimeout` in milliseconds? If yes, maybe we can reduce that to about `4000`

Reply:
By default, cypress takes screenshots and screencast for each test file and outputs it to a default folder. The settings are to specify the folder for the outputs when tests are run.
The default` pageLoadTimeout` is `6000` and sometimes that won't be enough to load all the elements in the page to test and that will make the test fail.

Reply:
Cool. While doing Agile, you normally incorporate exactly the changes that are needed until the need for extra changes arises. At this point, the question is do we need these extra settings? Bear in mind that the settings we are committing here is what will be run on circle ci. Does circle ci need to have screenshots and video assets?

I can also see you set the video option to false. What is the default for that? If video is false by default, no need to add that. And also, if the video is false, do we still need to have the `videosFolder` setting?

For the `pageLoadTimeout` setting, the question right now is are any specs failing presently because that setting was not added?

If the answer to all of the above is no, then we don't need the extra settings for now until the need arises. Make sense?

Reply:
Ok. For now, I don't think the need for video is needed and also the video option by default is true so I have to add the settings to make it false. For the `pageLoadTimeout`, anything less than `6000` will make the spec fail.